packages/web-app-files/src/store/actions.ts

```diff
@@ -125,7 +125,9 @@ export default {
         }
 
         return Promise.all(loadingResources).then(() => {
+          const currentFolder = context.getters.currentFolder
           context.commit('UPSERT_RESOURCES', fetchedResources)
+          context.commit('LOAD_INDICATORS', currentFolder.path)
```

Comment:
is it possible that the `currentFolder` var can be nullish?

Reply:
I thought about it, but I don't see a scenario where we don't have a folder to paste into. Do you?

Reply:
True. And now that I looked at it in detail - the currentFolder is already used at the top of the function. So yeah, fine like that 😅